routes/farmer/farmer.js

```diff
@@ -434,6 +458,50 @@ exports.updateMembership = function(req, res) {
         });
     }
 };
+/*
+exports.updateCommodity = function(req, res) {
```

Comment:
@nick22891 The commented code should be removed before the code is pushed. Helps reduce the confusion in the code review process.